doc/examples/ex51/ex51.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# GMT EXAMPLE 51
+#
+# Purpose:	Illustrate usage of OpenStreetMap coastlines in GMT
+# GMT modules:	convert, select, coast, plot
+# GDAL progs:	ogr2ogr
+
+
+# First we convert the shapefile to something more GMT friendly. Here we make
+# into a large ASCII file. Human-readable but not very efficient. GDALs ogr2ogr
+# is used with the "-F GMT" option indicating that the output should have the
+# native GMT format. Input is the downloaded land_polygons.shp, and the output
+# is named land_polygons_osm_planet.gmt. The resulting file is very large 
+# (>1 GB) but already usable. But wait, there is more ...
+
+# ogr2ogr -F GMT land_polygons_osm_planet.gmt land_polygons.shp
```

Comment:
```suggestion
ogr2ogr -f OGR_GMT land_polygons_osm_planet.gmt land_polygons.shp
```
1. Uncomment this line is OK, as we'll skip the example in GMT's nightly test
2. Although `-F GMT` may work for you. The ogr2ogr usage tells me to use `-f OGR_GMT` instead. I don't test it. Could you verify if it works for you?

Reply:
I ran both versions and in both cases it works as expected - yours as well as mine:

```
mbp15-kristof:OSM-Kueste kristof$ ogr2ogr -f OGR_GMT land_polygons_osm_planet.gmt land_polygons.shp
Warning 1: The output driver does not seem to natively support Integer64 type for field FID. Converting it to Real instead. -mapFieldType can be used to control field type conversion.
mbp15-kristof:OSM-Kueste kristof$ ogr2ogr -F GMT land_polygons_osm_planet.gmt land_polygons.shp
Warning 1: The output driver does not seem to natively support Integer64 type for field FID. Converting it to Real instead. -mapFieldType can be used to control field type conversion.
mbp15-kristof:OSM-Kueste kristof$
```

the lower-case `-f` is the proper switch but I was unable to find anything about `GMT` vs `OGR_GMT`. 

See
https://gdal.org/programs/ogr2ogr.html#cmdoption-ogr2ogr-f
https://gdal.org/drivers/vector/gmt.html 

So maybe `-f GMT` is the *right* variant?

Reply:
Ok, found `OGR_GMT` as valid `format_name` in `ogr2ogr --long-usage`. So `OGR_GMT` it is.

Reply:
`ogr2ogr --long-usage` lists `OGR_GMT`, not `GMT`.

`-f GMT` also works for me (GDAL 2.4.4). I'm OK with `-f GMT`.

Reply:
We seem to have replied almost at the same time. I'll take your `OGR_GMT` as the correct version as there is a chance that `GMT` is not recognized and therefore the proper format is guessed from the file extension as per https://gdal.org/programs/ogr2ogr.html#cmdoption-ogr2ogr-f . I think it is better to use documented file formats.